sdk/contentsafety/azure-ai-contentsafety/azure/ai/contentsafety/_operations/_patch.py

```diff
@@ -18,3 +22,82 @@ def patch_sdk():
+        :return: AnalyzeTextResult. The AnalyzeTextResult is compatible with MutableMapping
+        :rtype: ~azure.ai.contentsafety.models.AnalyzeTextResult
+        :raises ~azure.core.exceptions.HttpResponseError:
+        """
+        ...
+
+    def analyze_text(self, body: str, **kwargs: Any) -> _models.AnalyzeTextResult:
+        options = _models.AnalyzeTextOptions(text=body)
+        return super().analyze_text(body=options, **kwargs)
+
+    @overload
+    def analyze_image(
+        self, blob_url: str, *, content_type: str = "application/json", **kwargs: Any
+    ) -> _models.AnalyzeImageResult:
+        """Analyze Image.
+
+        A synchronous API for the analysis of potentially harmful image content. Currently, it supports
+        four categories: Hate, SelfHarm, Sexual, and Violence.
+
+        :param blob_url: The blob url of the image. Required.
+        :type blob_url: string
+        :keyword content_type: Body Parameter content-type. Content type parameter for JSON body.
+         Default value is "application/json".
+        :paramtype content_type: str
+        :keyword bool stream: Whether to stream the response of this operation. Defaults to False. You
+         will have to context manage the returned stream.
+        :return: AnalyzeImageResult. The AnalyzeImageResult is compatible with MutableMapping
+        :rtype: ~azure.ai.contentsafety.models.AnalyzeImageResult
+        :raises ~azure.core.exceptions.HttpResponseError:
+        """
+
+    @overload
+    def analyze_image(
+        self, image_content: bytes, *, content_type: str = "application/json", **kwargs: Any
+    ) -> _models.AnalyzeImageResult:
+        """Analyze Image.
+
+        A synchronous API for the analysis of potentially harmful image content. Currently, it supports
+        four categories: Hate, SelfHarm, Sexual, and Violence.
+
+        :param image_content: The Base64 encoding of the image. Required.
+        :type image_content: bytes
+        :keyword content_type: Body Parameter content-type. Content type parameter for JSON body.
+         Default value is "application/json".
+        :paramtype content_type: str
+        :keyword bool stream: Whether to stream the response of this operation. Defaults to False. You
+         will have to context manage the returned stream.
+        :return: AnalyzeImageResult. The AnalyzeImageResult is compatible with MutableMapping
+        :rtype: ~azure.ai.contentsafety.models.AnalyzeImageResult
+        :raises ~azure.core.exceptions.HttpResponseError:
+        """
+
+    def analyze_image(self, data: Union[str, bytes], **kwargs: Any) -> _models.AnalyzeImageResult:
+        _image = None
+        if isinstance(data, bytes):
+            _image = _models.ImageData(content=data)
+        else:
+            _image = _models.ImageData(blob_url=data)
+        options = _models.AnalyzeImageOptions(image=_image)
+        return super().analyze_image(body=options, **kwargs)
```

Comment:
What if users input `_models.AnalyzeImageOptions(...)` directly?

Reply:
the same as above.